gateway/setup/create_certs.sh

```diff
@@ -0,0 +1,19 @@
+if ! command -v openssl &> /dev/null
+then
+    echo "openssl could not be found"
+    exit
+fi
+mkdir -p ../certs
+cd ../certs
+# create CA private key
+openssl genrsa -des3 -out ca_key.pem 2048
+# create CA certificate
+openssl req -x509 -new -nodes -key ca_key.pem -sha256 -days 365 -out ca_cert.pem
+# create server private key
+openssl genrsa -des3 -out server_key.pem 
+# create CSR
+openssl req -new -key server_key.pem -out csr.pem -subj "/C=/ST=/L=/O=/CN=localhost"
```

Comment:
You can just put `"-subj "/CN=Local Hummingbot Gateway"` and avoid a bunch of "req: No value provided for xxx" error messages.